src/pages/api/_fake/complete_connect_webview.ts

```diff
@@ -32,8 +32,9 @@ export default withRouteSpec({
   })
 
   const relevant_cs = req.db.client_sessions.find((cs) =>
-    cs.connect_webview_ids.includes(connect_webview.connect_webview_id),
+    cs.workspace_id.includes(connect_webview.workspace_id),
```

Comment:
The original logic was not working because the client session actually never got updated with the `connect_webview_id`. I'm not sure how that was working in the first place and I think it was skipping over the `relevant_cs` most of the time.